generators/server/index.js

```diff
@@ -1024,6 +1024,31 @@ module.exports = JhipsterServerGenerator.extend({
                     this.copy(SERVER_MAIN_RES_DIR + 'config/cql/changelog/_insert_default_users.cql', SERVER_MAIN_RES_DIR + 'config/cql/changelog/00000000000000_insert_default_users.cql');
                 }
             }
+
+            if (this.applicationType === 'uaa') {
+                const keyStoreFile = SERVER_MAIN_RES_DIR + 'keystore.jks';
+                if (this.fs.exists(keyStoreFile)) {
+                    this.log(chalk.cyan('\nKeyStore \'' + keyStoreFile + '\' already exists. Leaving unchanged.\n'));
+                } else {
+                    shelljs.mkdir('-p', SERVER_MAIN_RES_DIR);
+                    var exitCode = shelljs.exec('keytool '+
```

Comment:
couldn't this be done asynchronously? I mean we need not really wait for this to complete right?